src/App.tsx

```diff
@@ -8,13 +9,24 @@ import {
   useDeskproAppEvents,
 } from "@deskpro/app-sdk";
 import { isNavigatePayload } from "./utils";
-import { LoadingAppPage } from "./pages";
+import { useLogout } from "./hooks";
+import {
+  LinkPage,
+  HomePage,
+  LoginPage,
+  LoadingAppPage,
+  AdminCallbackPage,
+} from "./pages";
 import type { FC } from "react";
 import type { EventPayload } from "./types";
 
 const App: FC = () => {
   const navigate = useNavigate();
+  const { pathname } = useLocation();
   const { client } = useDeskproAppClient();
+  const { logout, isLoading: isLoadingLogout } = useLogout();
+  const isAdmin = useMemo(() => pathname.includes("/admin/"), [pathname]);
+  const isLoading = [isLoadingLogout].some((isLoading) => isLoading);
```

Comment:
what are we doing here?

Reply:
This is a variable to show the spinner. Now I show only when the logout is going on. In the future, there will be when unlinking issue from the ticket. Maybe something else will be

Reply:
Yes but right now its only isLoadingLogout, so that should be removed
you can add a comment underneath it to keep that code in case you add another loading var

Reply:
After one PR I'll add one more variable - it's 100% (in this story https://app.shortcut.com/deskpro/story/119785/bitbucket-issue-details)